### PR TITLE
Fix/icon

### PR DESCRIPTION
**Pull request by arashmanteghi**

### CODE REVIEW

File: src/components/card/event-card/index.js [hunk 1]
```diff
@@ -75,18 +74,18 @@ const EventCard = (props: Props) => {
             <ShowDate date={date} color="gray" fontSize="12" />
             {
               isBookmarked
-                ? <BookmarkIcon type="bookmark" size="lg" color="gray" onClick={handleClickBookmark} />
-                : <BookmarkIcon type="bookmark-border" size="lg" color="gray" onClick={handleClickBookmark} />
```
Comment (marzzy) [inline src/components/card/event-card/index.js]: isBookmarked condition can be set only for type property cause the rest are the same

Comment (arashmanteghi) [reply src/components/card/event-card/index.js]: U R right. I'll consider it.

File: src/components/card/event-card/index.style.js [hunk 2]
```diff
@@ -31,10 +31,6 @@ export const Title = styled(Heading)`
   text-overflow: ellipsis;
 `;
 
-export const PlacePriceIcon = styled(Icon)`
```
Comment (marzzy) [inline src/components/card/event-card/index.style.js]: we also can add this feature for BookmarkIcon on this file line 22.

File: src/components/card/event-card/index.js [hunk 3]
```diff
@@ -113,18 +112,18 @@ const EventCard = (props: Props) => {
           <ShowDate date={date} color="gray" fontSize="12" />
           {
             isBookmarked
-              ? <BookmarkIcon type="bookmark" size="lg" color="gray" onClick={handleClickBookmark} />
-              : <BookmarkIcon type="bookmark-border" size="lg" color="gray" onClick={handleClickBookmark} />
+              ? <BookmarkIcon name="bookmark" size="lg" color="gray" onClick={handleClickBookmark} />
```
Comment (marzzy) [inline src/components/card/event-card/index.js]: isBookmarked condition can be set only for type property cause the rest are the same

**Pull request merged**